Parent/Parent/Dashboard/Calendar/CalendarViewController.swift

```diff
@@ -111,6 +116,7 @@ class CalendarViewController: UIViewController, CalendarDaysDelegate {
         selectedDate = date
         updatePage()
         clearPageCache()
+        delegate?.selectedDateDidChange(selectedDate)
```

Comment:
You will also need to call the delegate on line 166ish since paging changes the selected date too.

Reply:
should that just call `setSelectedDate(` ?

Reply:
the problem is that if it tries to clear the page cache we get a crash in UIPageViewController